docker-compose.local.yaml

```diff
@@ -18,6 +18,9 @@ services:
       - ./packs:/packs
       - ${PWD}/lib/public/en_about.html:/bento-public/src/public/en_about.html
       - ${PWD}/lib/public/fr_about.html:/bento-public/src/public/fr_about.html
+      - ${PWD}/lib/public/branding.png:/bento-public/src/public/branding.png
```

Comment:
corrrect path appears to be `/bento-public/src/public/assets/branding.png`, missing `/assets/`

Reply:
Yes, correct it